### PR TITLE
[NR-331312] Add AEI session mapper

### PR DESCRIPTION
**Pull request by cthomas-newrelic**
Acceptance criteria:

* a PID-to-session map will be created and persisted in local AEI cache
* for AEI events gathered during the first agent run, the sessionID of the event will be the one that recorded the event
* for AEI events gathered in successive sessions, a new attribute will be added to provide the session ID in which the AEI occurred 
  * in this case session ID will never match recorded session ID
* session ID map will be updated each time the agent starts
* session ID map will be updated each time the agent restarts
* session ID map will be pruned when the set of AEI records returned by the ART no longer includes elements in the map
* an attribute with the historic session ID will be inserted into the event as `sessionId`.

Tests will be created to reflect this criteria.

### CODE REVIEW

**Comment by cthomas-newrelic** [comment]
Shoot, this test fix is in another PR.  I can port it to this branch, or wait until the other PR is merged.

**Review by ndesai-newrelic — APPROVED** [review]

**Pull request merged**
